apps/src/code-studio/components/ExportDialog/Dialog.jsx

```diff
@@ -73,122 +51,35 @@ const styles = {
     fontSize: 13,
     fontWeight: 'bold'
   },
-  uploadIconButton: {
-    ...baseStyles.button,
-    backgroundColor: color.default_blue,
-    color: color.white
-  },
-  iosAppStoreButton: {
-    ...baseStyles.button,
-    backgroundColor: color.purple,
-    color: color.white
-  },
-  androidGooglePlayButton: {
-    ...baseStyles.button,
-    backgroundColor: color.purple,
-    color: color.white
-  },
   cancelButton: {
-    ...baseStyles.button,
+    ...commonStyles.button,
     backgroundColor: color.gray,
     color: color.black
   },
   actionButton: {
-    ...baseStyles.button,
+    ...commonStyles.button,
     backgroundColor: color.orange,
     color: color.white
   },
   actionButtonDisabled: {
-    ...baseStyles.button,
+    ...commonStyles.button,
     backgroundColor: color.gray,
     color: color.white
   },
   backButton: {
-    ...baseStyles.button,
+    ...commonStyles.button,
     backgroundColor: color.gray,
     color: color.black
   },
   backButtonDisabled: {
-    ...baseStyles.button,
+    ...commonStyles.button,
     backgroundColor: color.gray,
     color: color.white
   },
-  p: {
-    ...baseStyles.text
-  },
```

Comment:
nice cleanup 
(some of these comments are for my own notes. I'll try to make it obvious if any action is needed)